src/blog/2024/10/building-prometheus-dashboard-with-node-red.md

```diff
@@ -0,0 +1,266 @@
+---
+title: Using Prometheus in Node-RED for monitoring 
+subtitle: Visualizing Metrics Effortlessly with Node-RED and Prometheus
+description: Learn to integrate Prometheus with Node-RED for monitoring and exporting metrics and create dashboards to visualize them.
+date: 2024-10-12
+authors: ["sumit-shinde"]
+image:
+tags:
+   - node-red
+   - nodered prometheus 
+   - node-red timeseries database
+   - node-red prometheus
+   - system monitoring using prometheus
+---
+
+Prometheus is a popular tool for monitoring and collecting system metrics. It helps you track server performance, resource usage, and application health. It’s widely used in various applications, including IoT, where it can monitor data such as temperature and humidity. By collecting these metrics, you can gain valuable insights into your systems and ensure they run smoothly.
```

Comment:
```suggestion
Prometheus is a popular tool for monitoring and collecting server metrics. It helps you track performance, resource usage, and application health. By collecting these metrics, you can gain valuable insights into your systems and ensure they run smoothly.
```

I've not seen widespread use at all, it's not really well suited for IoT either. What's the source?